### DISCUSSION TITLE
Version 11.6.0 not backward compatible for the accept option

### DISCUSSION THREAD

**Discussion by Seijinx** [react-dropzone/react-dropzone#1130] [General]
I have the feeling `11.6.0` should have been a `12.0.0` since it breaks the `accept` option for `useDropzone` no more supporting extensions filtering.

For now I have to freeze to `11.5.3` to delay an important refactor on my side.

What do you think ?

**Comment by rolandjitsu** [comment]
I think you might be right @Seijinx . While the feature addition was not intended as a breaking change, the `accept` does work differently when the file system access API is supported.

I'll have to see how to undo or fix the versioning.

**Reply by Seijinx** [reply]
Maybe a PATCH (`11.7.1`) adding a default option to "opt-out" of the FileSystem API at first could be a great workaround without undoing changes ?

BTW. The work done around FileSystem API is huge and important for the evolution of react-dropzone, thank you for this!

**Reply by rolandjitsu** [reply]
I was actually thinking about adding a feature to not use it for the users that don't want it. So I think I'll do just that. And probably I'll make a breaking change to enable by default instead of disable. I'll do the changes tomorrow as it's late here.

> BTW. The work done around FileSystem API is huge and important for the evolution of react-dropzone, thank you for this!

Thanks. Hopefully the API will become living standard 🤞  And whatever other suggestions for improvement you have, please feel free to create a new issue or start a discussion!

**Reply by rolandjitsu** [reply]
Ok, v11.7.1 was just published. Let me know if that fixes the issue.

**Reply by Seijinx** [reply]
@rolandjitsu Great work, no regression on my part. These new defaults do not breaks my setup.

Thank you for this swift intervention. Always appreciated!
